l10n_ch_import_camt054/models/account_bank_statement.py

```diff
@@ -0,0 +1,10 @@
+
+from odoo import models, fields
+
+class AccountBankStatement(models.Model):
+    _inherit = 'account.bank.statement.line'
```

Comment:
Either you made a mistake and it should be 'account.bank.statement' instead of line, or you are inheriting bank_statement_line and should therefore move your function `_prepare_reconciliation_move_line` in the bank_statement_line.py file (and remove this file)